generic/tests/iozone_windows.py

```diff
@@ -26,10 +26,13 @@ def run(test, params, env):
     results_path = os.path.join(test.resultsdir,
                                 'raw_output_%s' % test.iteration)
     analysisdir = os.path.join(test.resultsdir, 'analysis_%s' % test.iteration)
+    device_key = params.get("device_key", "VolumeName")
+    device_value = params.get("device_value", "System")
```

Comment:
Hi @suqinhuang 

I have not found a VolumeName called "System", could you show me more details here?